src/main/scala/Domain/Door.scala

```diff
@@ -56,7 +56,7 @@ object Door {
       }
 
   val aggregateLogic: List[Flow[Unit]] = List(
-    handler { case Register(id) => emitEvent(Registered(id)) } >> waitFor { case Registered(_) => () },
+    handler { handler[Register, Registered] } >> waitFor { case Registered(_) => () },
```

Comment:
would a different name for the new `handler` fit better though? for not to conflict (mentally) with the existing function that lifts to the free monad

a few ideas that come to my mind are `noValidation`, `direct`, something else?